pkgs/os-specific/linux/wireguard/default.nix

```diff
@@ -1,11 +1,16 @@
-{ stdenv, kernel, wireguard-tools, perl }:
+{ stdenv, fetchzip, kernel, perl, wireguard-tools }:
 
 # module requires Linux >= 3.10 https://www.wireguard.io/install/#kernel-requirements
 assert stdenv.lib.versionAtLeast kernel.version "3.10";
```

Comment:
Am I missing any reason why you didn't add a `versionOlder kernel.version "5.6"` assertion?

Reply:
I wasn't sure sure that's appropriate since the 5.6 kernel isn't included in `nixpkgs` yet. Very happy to add if you think it's the right thing to do.